Karen_dinner_party.rb

```diff
@@ -0,0 +1,65 @@
+
+#use for a pre-populated party with guest, food, and bevs. Otherwise use user input option below.
+#party_elements = {beverages: ["coke", "diet coke", "sprite", "tequila", "tequiza", "zima", "bud light lime", "orange juice", "water", "coffee", "tea", "jim beam", "cerveza"], food: ["porkchop", "veggie nuggets", "shrimp", "spaghetti"], people:
+#["Karen", "Wolfie", "Erin", "Tim", "Thom", "Stephen", "Ken", "James"], seat_num:[]}
+
+#user input arrays
+party_elements = {beverages: [], food: [], people:
+[], seat_num:[]}
+
+
+#puts "#{party_elements[:people].sample} will have the #{party_elements[:food].sample} to eat with #{party_elements[:food].sample} on the side. And to drink, #{party_elements[:beverages].sample}! Yum!"
+
+puts "Who would you like to invite to the party?"
+  array = party_elements[:people]
```

Comment:
Super nit-picky, but it's best to avoid naming things something like `array`, even just adding what is in the array to the variable name is helpful. Like `people_array`.